.github/workflows/release.yaml

```diff
@@ -202,7 +220,9 @@ jobs:
       - uses: richb-hanover/cargo@v1.1.0
         with:
           command: release
-          args: publish --no-confirm -x
+          args:
+            publish --no-confirm ${{ github.event.inputs.dry-run == 'true' && ''
```

Comment:
FYI I _think_ the `''` will evaluate to false and `'--execute'` would always be returned!

GHA syntax is kinda dangerous there...

Reply:
> GHA syntax is kinda dangerous there...

(but given the number of jobs, I think this is the correct design, relative to a `task release` (which only would work in GHA anyway given the credentials)